index.js

```diff
@@ -702,20 +702,93 @@
     }
 
     function categorizeCommitMessage(message) {
+        message = message.trim();
         const categoryResult = categoryRegex.exec(message);
         if (!categoryResult)
             return ["Uncategorized"];
 
         const categoryString = categoryResult[1];
         let categories = [];
         if (categoryString.indexOf("+") >= 0)
-            categories.push(...categoryString.split("+"));
+            categories.push(...categoryString.split("+").map(c => c.trim()));
         else
-            categories.push(categoryString);
+            categories.push(categoryString.trim());
 
         return categories;
     }
 
+    class Category extends Array {
```

Comment:
It doesn't look like anything uses `Category` as an `Array` now.
```suggestion
    class Category {
```